include/CppUTest/CommandLineTestRunner.h

```diff
@@ -49,8 +50,12 @@ class CommandLineTestRunner
 
     static int RunAllTests(int ac, const char** av);
     static int RunAllTests(int ac, char** av);
+    static int RunAllTests(int ac, const wchar_t** av);
+    static int RunAllTests( int ac, wchar_t** av );
```

Comment:
For symmetry I think it would be best, although the char one might just be there for historical reasons that it was the first 'RunAllTests' function.